components/brave_wallet_ui/constants/types.ts

```diff
@@ -795,6 +797,9 @@ export enum WalletRoutes {
   // send
   Send = '/send',
 
+  // dev bitcoin screen
+  DevBitcoin = '/dev-bitcoin',
```

Comment:
Is `dev-bitcoin` `page` enabled by default?

Reply:
Needs bitcoin feature to be enabled.
https://github.com/brave/brave-core/pull/17978/files#diff-af0cc8816d323a308707bc347f56b9368f8b5e05152174c35b07068bb88908c9R291

Reply:
Can we have the `/dev-bitcoin` page accessible only in staging/nightly/dev builds?

Reply:
@onyb, is it possible?
And as I mentioned in description I can totally cut this dev bitcoin ui from PR and keep it local(or maybe move to new wallet-internals ui)

Reply:
@thypon 
Is enough that that page is hidden by default and available only when user runs browser with command line flag `--enable-features=BraveWalletBitcoin` ?

Reply:
Perfectly fine, yup!